Description: Use compiler from debian's golang package instead of vendored binary
Author: Andres Salomon <dilinger@debian.org>


--- a/third_party/dawn/tools/generate-sources-gn.py
+++ b/third_party/dawn/tools/generate-sources-gn.py
@@ -40,8 +40,7 @@
 
 
 def main() -> int:
-    go_binary = os.path.join(DAWN_ROOT, 'tools', 'golang',
-                             cipd_deps.get_cipd_platform(), 'bin', 'go')
+    go_binary = '/usr/bin/go'
     if sys.platform == 'win32':
         go_binary += '.exe'
 
